django_multitenant/mixins.py

```diff
@@ -37,7 +37,8 @@ def wrap_many_related_manager_add(many_related_manager_add):
     """
 
     def add(self, *objs, through_defaults=None):
-        if get_current_tenant():
+
+        if hasattr(self.through, "tenant_field") and get_current_tenant():
```

Comment:
That is the actual fix. Checking if transition table has tenant_id, if not, we are not trying to add tenant id into transition table